### PR TITLE
Add Community Contribution for File Ingestion Tutorial

### PR DESCRIPTION
**Pull request by northernsquare**

### CODE REVIEW

File: file_ingestions.Rmd [hunk 1]
```diff
@@ -0,0 +1,93 @@
+```{r, include=FALSE}
```
Comment (philkim72) [inline file_ingestions.Rmd:1]: The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

The second line should be blank, followed by the author name(s).

**Review by philkim72 — CHANGES_REQUESTED** [review]
The first line should start with a single hashtag #, followed by a single whitespace, and then the title.

The second line should be blank, followed by the author name(s).

**Comment by philkim72** [comment]
I requested changes 9 days ago.  Please make chances so that I can merge the pull request.

**Comment by jtr13** [comment]
Thanks, I'll handle.

**Comment by northernsquare** [comment]
@jtr13 Updated formatting accordingly - please review when you get a chance :)

**Comment by philkim72** [comment]
thank you for your contribution! :tada:

**Pull request merged**